content/docs/use-cases/sharing-data-and-model-files.md

```diff
@@ -1,99 +0,0 @@
-# Sharing Data and Model Files
-
-Like Git, DVC allows for a distributed environment and collaboration. We make it
-easy to consistently get all your data files and directories into any machine,
-along with matching source code. All you need to do is to setup
-[remote storage](/doc/command-reference/remote) for your <abbr>DVC
-project</abbr>, and push the data there, so others can reach it. Currently DVC
-supports Amazon S3, Microsoft Azure Blob Storage, Google Drive, Google Cloud
-Storage, SSH, HDFS, and other remote locations. The list is constantly growing.
-(For a complete list and configuration instructions, refer to `dvc remote add`.)
-
-![](/img/model-sharing-digram.png) _Data is stored in storage you control,
-separate from code._
```

Comment:
I would perhaps consider moving this image to the UC index for now (https://dvc.org/doc/use-cases). WDYT @shcheklein ?

Reply:
It seems too specific to me still to be honest

Reply:
OK well so far this PR only removes the page (and image) as agreed.